src_erl/erlBridge/opennnWorkerDefintions.hrl

```diff
@@ -0,0 +1,7 @@
+
+
+getEnumMapActivationFunction() -> #{enum_threshold => 1, enum_SymmetricThreshold => 2, c=> 4, "a" => 1, "b" => 2, "c" => 4},.
```

Comment:
What is this? a b c? 
It might be an example or something - it shouldn't find its way to master
@kapelnik @evgenyan95

Reply:
done